modules/nw-operator-cr.adoc

```diff
@@ -52,7 +52,7 @@ The CNO configuration inherits the following fields during cluster installation
 ifdef::post-install-network-configuration,operator[]
 [NOTE]
 ====
-After cluster installation, you cannot modify the fields listed in the previous section.
+After cluster installation, you can only modify the `clusterNetwork` IP address range. The default network type can only be changed from OpenShift SDN to OVN-Kubernetes through migration.
```

Comment:
Link to the migration docs?

Also, looking at the preview, the text right before the list of fields says "these fields cannot be changed:" so that probably needs an update too

Reply:
We unfortunately can't add xref links in modules at this time, but I did add the link to the "additional resources" section